.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 .tmp
+bazel-*
```

Comment:
add a new line at the end of the file, plz.

Reply:
added the new line, thx for the review